tools/docker-format/Dockerfile

```diff
@@ -1,8 +1,28 @@
 FROM ubuntu:cosmic
 
+# repositories are obsolete. the image need to be changed.
```

Comment:
If we update to something newer than cosmic, do we also get golang>=1.12?

Reply:
Upgrading from cosmic to focal ( 20.04 LTS ) gives you exactly golang 1.13. 
It requires python3 but the same cmake-format package version works. 
The default clang-format is 10.0 and needs to be downgraded not to have side effects on the current c++ code.

On other hand, installing golang manually makes the upgrade path clear should buildifier requirements change again.

currently in this PR
```
docker run -v $PWD:/opencensus-cpp -it opencensus-cpp/format:cosmic /opencensus-cpp/tools/docker-format/run.sh
clang-format version 7.0.0-3 (tags/RELEASE_700/final)
Running buildifier.
Running cmake-format 0.6.10.
```

potential "focal" version
```
docker run -v $PWD:/opencensus-cpp -it opencensus-cpp/format:focal /opencensus-cpp/tools/docker-format/run.sh

clang-format version 7.0.1-12 (tags/RELEASE_701/final)
Running buildifier.
Running cmake-format 0.6.10.
```

I can submit the latter if you'd rather.

Reply:
1. Is the "focal" version simpler?

2. Instead of installing golang in "cosmic" is it easier to just install a release of buildifier? e.g. https://github.com/bazelbuild/buildtools/releases/download/3.3.0/buildifier